feconf.py

```diff
@@ -257,6 +257,10 @@
 # schema changes.
 CURRENT_PLATFORM_PARAMETER_RULE_SCHEMA_VERSION = 1
 
+# This value is used to initialize the version number for the versioned
```

Comment:
This should describe what the value is, not how it's used. E.g.: ``Version number that should be used when initializing versioned entities when no specific version is known.`` or something similar.

Reply:
Done!